MIGRATION.MD

```diff
@@ -0,0 +1,267 @@
+# Migration from 1.x to 2.0
+
+The main changes introduced in SDK 2.0 compared to 1.x are:
+
+1. All relevant products (RUM, Trace, Logs, etc.) are now extracted into different modules. That allows you to integrate only what is needed into your application.
+
+Whereas all products in version 1.x were contained in the single artifact `com.datadoghq:dd-sdk-android:x.x.x`, you now need to adopt the following artifacts instead:
+
+* RUM: `com.datadoghq:dd-sdk-android-rum:x.x.x`
+* Logs: `com.datadoghq:dd-sdk-android-logs:x.x.x`
+* Trace: `com.datadoghq:dd-sdk-android-trace:x.x.x`
+* WebView Tracking: `com.datadoghq:dd-sdk-android-webview:x.x.x`
+* OkHttp instrumentation: `com.datadoghq:dd-sdk-android-okhttp:x.x.x`
+
+**Note**: In case of NDK Crash Reporting and WebView Tracking usage it's also needed to add RUM and/or Logs artifacts to be able to report events to RUM and/or Logs respectively.
+
+Reference to the `com.datadoghq:dd-sdk-android` artifact should be removed from your Gradle buildscript, this artifact doesn't exist anymore.
+
+All other artifacts Maven coordinates stay the same.
```

Comment:
```suggestion
All other artifacts, including Maven coordinates, stay the same.
```

Reply:
I'm not sure about the intended message here, but it seems like an incomplete sentence and I guessed what you meant..

Reply:
I wanted to say that the Maven coordinates of all the other artifacts stay the same, so I guess the suggestion should be updated.